static/js/https/domains.js

```diff
@@ -67,7 +71,7 @@ $(document).ready(function () {
       return ""
     else
       return "" +
-        "<a href=\"" + labsUrlFor(row.domain) + "\" target=\"blank\">" +
+        "<a href=\"" + labsUrlFor(row.canonical) + "\" target=\"blank\">" +
```

Comment:
SSL Labs has changed their behavior to no longer auto-detect whether a domain is meant to be accessed at `www` or the root of the domain. Since we go to great lengths to figure that out during our `inspect` / `pshtt` scans, we can use that data here to send SSL Labs the endpoint we wish it to scan.